docs/rules.md

```diff
@@ -19,9 +19,10 @@ To update it edit `docs/rules.md` in the
 | :x: | warning | called_dangerous_global | | `%s` called in potentially dangerous manner' | | | | |
 | :white_check_mark: | error? | eval | | In order to prevent vulnerabilities, the `setTimeout` 'and `setInterval` functions should be called only with function expressions as their first argument. | | [testcases/javascript/actions.py](https://github.com/mozilla/amo-validator/blob/7a8011aba8bf8c665aef2b51eb26d0697b3e19c3/validator/testcases/javascript/actions.py#L488) | | EVAL_STRING_ARG |
 | :x: | warning | low_level_module (not from src) | | Usage of low-level or non-SDK interface | | | | |
-| :x: | warning | widget | | Use of deprecated SDK module | | | | |
+| :white_check_mark: | warning | widget | | Use of deprecated SDK module | | | | DEPREC_SDK_MOD_WIDGET |
```

Comment:
No old code for this one? Adding `null` like you did below is nice to indicate that at least :smile: